src/code/graph.c

```diff
@@ -179,6 +179,7 @@ void Graph_TaskSet00(GraphicsContext* gfxCtx) {
     {
         CfbInfo* cfb;
 
+        // Schedule 3 second RCP timeout
```

Comment:
```suggestion
        // Schedule RCP timeout after 3 seconds
```
?

Reply:
hmm makes it sound like the scheduling happens in 3 seconds instead of now

Reply:
`after` -> `for` ?

Reply:
that would address that concern, but what's wrong with "Schedule 3 second RCP timeout"?

Reply:
I borrowed the wording from some other places OS_USEC_TO_CYCLES is used for this purpose